rocAL_pybind/examples/rocAL_api_audio_unittest.py

```diff
@@ -0,0 +1,84 @@
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import random
+import numpy as np
+from amd.rocal.plugin.pytorch import ROCALAudioIterator
+import torch
+# torch.set_printoptions(threshold=10_000)
+np.set_printoptions(threshold=1000, edgeitems=10000)
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import math
+# import rocal_pybind.tensor
+import sys
+import cv2
+import matplotlib.pyplot as plt
+import os
+def draw_patches(img, idx, device):
+    #image is expected as a tensor, bboxes as numpy
+    import cv2
+    image = img.detach().numpy()
+    audio_data = image.flatten()
+    label = idx
+    # label = idx.cpu().detach().numpy() #TODO: Uncomment after the meta-data is enabled
+    print("label: ", label)
+    # Saving the array in a text file
+    file = open("results/rocal_data_new"+str(label)+".txt", "w+")
+    content = str(audio_data)
+    file.write(content)
+    file.close()
+    plt.plot(audio_data)
+    plt.savefig("results/rocal_data_new"+str(label)+".png")
+    plt.close()
+def main():
+    if  len(sys.argv) < 3:
+        print ('Please pass audio_folder file_list cpu/gpu batch_size')
+        exit(0)
+    try:
+        path= "OUTPUT_IMAGES_PYTHON/NEW_API/FILE_READER/" + "audio"
+        isExist = os.path.exists(path)
+        if not isExist:
+            os.makedirs(path)
+    except OSError as error:
+        print(error)
+    data_path = sys.argv[1]
+    file_list = sys.argv[2]
+    if(sys.argv[3] == "cpu"):
+        _rali_cpu = True
+    else:
+        _rali_cpu = False
```

Comment:
Please check if it is better to add a comment stating GPU support is not yet given for audio
Otherwise it is implicit that user can try with both backends